frontend/src/main/components/Nav/Footer.js

```diff
@@ -5,11 +5,10 @@ export default function Footer() {
     <footer className="bg-light pt-3 pt-md-4 pb-4 pb-md-5" data-testid="Footer">
       <Container>
         <p>
-          This is a sample webapp using React with a Spring Boot backend. See
-          the source code on{" "}
-          <a href="https://github.com/ucsb-cs156/spring-react-template">
-            Github.
-          </a>
+          Built by students using React and Spring Boot ❤️ See the source code
+          on{" "}
+          <a href="https://github.com/ucsb-cs156-f24/proj-rec-f24-07">Github</a>
```

Comment:
```suggestion
          <a href="https://github.com/ucsb-cs156/proj-rec">Github</a>
```

Please change it to this one; this is the repo that will contain the code that gets passed down to the next instance of the course in S25.

Reply:
Done!